source/change-streams/tests/unified/change-streams.yml

```diff
@@ -0,0 +1,74 @@
+description: "change-streams"
+schemaVersion: "1.0"
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: []
+      ignoreCommandMonitoringEvents: []
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+initialData:
+  - collectionName: *collection0
+    databaseName: *database0
+    documents: []
+tests:
+  - description: "Test array truncation"
```

Comment:
When I last reviewed this PR, I thought there was consensus about not introducing a test that only asserted server behavior. Previous comments from [Jeff](https://github.com/mongodb/specifications/pull/909#issuecomment-778212888) and [Divjot](https://github.com/mongodb/specifications/pull/909#issuecomment-778315408) suggested this wouldn't be useful for Java and Go, respectively, since they both inspect change stream events as raw BSON instead of using their model classes. AFAIK, most drivers don't model change stream events, so I'm wondering if there are actually any other drivers that would benefit from this test.

Alternatively, did folks change their mind about it being generally useful to have a spec test that asserts server behavior?

Reply:
@stIncMale pointed out that we have some tests which assert server behavior. Also, even if there are only a handful of drivers that convert change stream results to native types, it seems useful to have this test rather than have each of those drivers figure out how to force the server to return `truncatedArrays` individually.

Reply:
@divjotarora contacted me in Slack and we agreed that I will try and create a unified test.

Speaking about the usefulness, it appears to me that different people who have participated in the discussion have different opinions:
- I do not see much usefulness from the test regardless of whether a driver's unified runner deserializes `ChangeStreamDocument`s or not. However, I also do not have anything agains having a unified test.
- @jyemin appears to see usefulness for drivers that deserialize `ChangeStreamDocument` in unified tests, and proposed (in Slack) a way to achieve this in the Java driver (has not been done; Jeff, I apologize if I interpreted you incorrectly).
- @divjotarora appears to lean towards having a test based on our dialog in Slack (again, I apologize if I interpreted that incorrectly).